cmd/logs/cmd.go

```diff
@@ -1,79 +1,82 @@
 package logs
 
 import (
+	"fmt"
 	"net/http"
 
 	"numerous.com/cli/cmd/args"
 	"numerous.com/cli/cmd/errorhandling"
 	"numerous.com/cli/cmd/group"
 	"numerous.com/cli/cmd/output"
+	"numerous.com/cli/cmd/usage"
 	"numerous.com/cli/internal/app"
 	"numerous.com/cli/internal/dir"
 	"numerous.com/cli/internal/gql"
 
 	"github.com/spf13/cobra"
 )
 
-var Cmd = &cobra.Command{
-	Use:     "logs [app directory]",
-	RunE:    run,
-	Short:   "Display running application logs",
-	GroupID: group.AppCommandsGroupID,
-	Long: `Read the logs of an application deployed to an organization on the
+const longFormat string = `Read the logs of an application deployed to an organization on the
 Numerous platform.
 
-If <name> and <organization> flags are set, they define the app to read logs
-from. If they are not, the default deployment section in the manifest is used,
-if it is defined.
+%s
+
+%s
+`
+const cmdActionText string = "to read logs from"
 
-If [app directory] is specified, that directory will be used to read the
-app manifest for the default deployment information.
+var long string = fmt.Sprintf(longFormat, usage.AppIdentifier(cmdActionText), usage.AppDirectoryArgument)
 
-If no [app directory] is specified, the current working directory is used.`,
-	Example: `To read the logs from a specific app deployment, use the following form:
+const example string = `To read the logs from a specific app deployment, use the following form:
 
     numerous logs --organization "organization-slug-a2ecf59b" --app "my-app"
 
 Otherwise, assuming an app has been initialized in the directory
 "my_project/my_app" and has a default deployment defined in its manifest:
 
     numerous logs my_project/my_app
-`,
-	Args: args.OptionalAppDir(&appDir),
+`
+
+var Cmd = &cobra.Command{
+	Use:     "logs [app directory]",
+	RunE:    run,
+	Short:   "Display running application logs",
+	GroupID: group.AppCommandsGroupID,
+	Long:    long,
+	Example: example,
+	Args:    args.OptionalAppDir(&cmdArgs.appDir),
 }
 
-var (
-	orgSlug    string
-	appSlug    string
+var cmdArgs struct {
+	appIdent   args.AppIdentifierArg
 	timestamps bool
-	appDir     string = "."
-)
+	appDir     string
```

Comment:
is it okay to remove the default here?

Reply:
It should be. Similar commands do not have a default value. The empty string works similar to dot for our purposes.

```go
filepath.Abs("")  # returns working directory and no error
filepath.Join("", "some", "relative", "path")  # returns "some/relative/path"

# and we have
filepath.Join("surrounded", ".", "surrounded") == filepath.Join("surrounded", "", "surrounded")
```